tv/tv.cpp

```diff
@@ -568,10 +574,15 @@ const llvm::Module * unwrapModule(llvm::Any IR) {
   llvm_unreachable("Unknown IR unit");
 }
 
+
 const char* skip_pass_list[] = {
   "{anonymous}::TVInitPass",
-  "ModuleToFunctionPassAdaptor<{anonymous}::TVFinalizePass>",
+  "{anonymous}::TVFinalizePass",
+  "AlwaysInlinerPass",
```

Comment:
the list of passes here should be only the ones used by clang. The `opt` from unit tests can be skipped automatically as described in my other comment.